cli/cmd/service/create.go

```diff
@@ -17,10 +24,65 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			res1, err := client.CreateService(cmd.Context(), &adminv1.CreateServiceRequest{
+			var attrs map[string]any
+			if attributes != "" {
+				if err := json.Unmarshal([]byte(attributes), &attrs); err != nil {
+					return fmt.Errorf("failed to parse --attributes as JSON: %w", err)
+				}
+			}
+
+			req := &adminv1.CreateServiceRequest{
 				Name:             args[0],
 				OrganizationName: ch.Org,
-			})
+			}
+
+			if ch.Interactive && orgRole == "" {
+				ok, err := cmdutil.ConfirmPrompt("Do you want to assign an organization role to the service?", "", false)
+				if err != nil {
+					return err
+				}
+				if ok {
+					err = cmdutil.SelectPromptIfEmpty(&orgRole, "Select role", orgRoles, "")
+					if err != nil {
+						return err
+					}
+				}
+			}
+
+			if ch.Interactive && projectRole == "" {
```

Comment:
This means that user will have to set `--interactive=false` if they want to use this cmd programatically.